src/main/groovy/com/avast/gradle/dockercompose/tasks/ComposeUp.groovy

```diff
@@ -164,6 +169,7 @@ class ComposeUp extends DefaultTask {
     Map<String, Object> getDockerInspection(String containerId) {
         new ByteArrayOutputStream().withStream { os ->
             project.exec { ExecSpec e ->
+                extension.setExecSpecWorkingDirectory(e)

```

Comment:
We execute just `docker` command that is current directory agnostic IMHO. So we could remove this line.